functional/platform/nspawn.go

```diff
@@ -302,14 +302,18 @@ func (nc *nspawnCluster) createMember(id string) (m Member, err error) {
 
 		// minimum requirements for running systemd/coreos in a container
 		fmt.Sprintf("mkdir -p %s/usr", fsdir),
+		//fmt.Sprintf("mkdir -p %s/lib", fsdir), //ubuntu
```

Comment:
I tried to run these tests in ubuntu without cloud-config. Decided to keep this. If you don't like - i'll remove.